src/Sylius/Bundle/AdminBundle/Resources/config/services/twig_component.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ This file is part of the Sylius package.
+  ~
+  ~ (c) Sylius Sp. z o.o.
+  ~
+  ~ For the full copyright and license information, please view the LICENSE
+  ~ file that was distributed with this source code.
+  -->
+
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <service id="Sylius\Bundle\AdminBundle\TwigComponent\Shared\Navbar\NotificationsComponent">
```

Comment:
If I will want to override this service as end-user, my services definition will look this this
```yaml
services:
    Sylius\Bundle\AdminBundle\TwigComponent\Shared\Navbar\NotificationsComponent:
        class: App\TwigComponent\Admin\Shared\Navbar\NotificationsComponent
```

What's the best practice here? To set a non-class service `id` or something else?